docs/release-notes.md

```diff
@@ -9,6 +9,22 @@ meta:
 ---
 # Release Notes
 
+## 7.5.0
+### New Single Sign-On Settings Page
+Owners of Organizations can now setup SSO for Insights through the UI. Setting up SSO requires a valid Metadata URL and Email Domain. Furthermore, Organizations
+will be able to enforce the use of SSO through Insights.
+
+### Snoozing of Action Items
+Users are now able to **Snooze** Action Items. This will set the Action Item Resolution to **Snoozed**. Users can take advantage of this feature for Action Items
```

Comment:
```suggestion
Users are now able to **Snooze** Action Items for a fixed period of time (1 month, 1 week, or 1 day). This will set the Action Item Resolution to **Snoozed**. Users can take advantage of this feature for Action Items
```